components/main_layout/header/view.coffee

```diff
@@ -97,7 +97,7 @@ module.exports = class HeaderView extends Backbone.View
     false
 
   openAuth: (options) ->
-    if sd.SHOW_ARTIST_CTA
+    if sd.NEW_ARTIST_PAGE_CTA is 'new_cta'
```

Comment:
Same as above, when we're in the new AB test group (and on an artist page), we want to change the behavior of the main header signup/login to a custom mediator event the CTA bar is listening for.